ch.theologeek.Manuskript.json

```diff
@@ -0,0 +1,147 @@
+        "directory": "texlive",
+        "subdirectories": true,
+        "autodelete": true,
+        "version": "21.08"
+      }
+  },
+  "command": "manuskript",
+  "finish-args": [
+	"--share=ipc",
+	"--socket=x11",
+	"--device=dri",
+	
+	/* access to local files */
+    "--filesystem=home",
+    "--env=QT_QPA_PLATFORM=xcb",
+    "--env=TMPDIR=/var/tmp",
+    
+    /* add paths of TeXlive Flatpak extension binaries */
+    "--env=PATH=/usr/bin:/app/bin:/app/texlive/bin:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux"
+  ],
+  "modules": [
+    "python3-modules.json",
+	{
+      "name" : "pyqt5",
+      "config-opts" : [
+        "--disable-static",
+        "--enable-x11"
+      ],
+      "sources" : [
+        {
+          "type" : "archive",
+          "url": "https://files.pythonhosted.org/packages/4d/81/b9a66a28fb9a7bbeb60e266f06ebc4703e7e42b99e3609bf1b58ddd232b9/PyQt5-5.14.2.tar.gz",
+          "sha256": "bd230c6fd699eabf1ceb51e13a8b79b74c00a80272c622427b80141a22269eb0"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "python3 configure.py --assume-shared --confirm-license --no-designer-plugin --no-qml-plugin --sysroot=/app --qsci-api --qsci-api-destdir=/app/qsci --sipdir=/app/share/sip --sip=/app/bin/sip --sip-incdir=/app/include QMAKE_CFLAGS_RELEASE='-I/usr/include/python3.9/' QMAKE_CXXFLAGS_RELEASE='-I/usr/include/python3.9/'"
+          ],
+          "dest-filename": "configure"
+        }
+      ],
+      "modules" : [
+        {
+          "name" : "sip",
+          "sources" : [
+            {
+              "type" : "archive",
+              "url" : "https://distfiles.macports.org/py-sip/sip-4.19.22.tar.gz",
+              "sha256" : "e1b768824ec1a2ee38dd536b6b6b3d06de27b00a2f5f55470d1b512306e3be45"
+            },
+            {
+              "type": "script",
+              "commands": [
+                "python3 configure.py --sip-module PyQt5.sip -b ${FLATPAK_DEST}/bin -d ${FLATPAK_DEST}/lib/python3.9/site-packages -e ${FLATPAK_DEST}/include -v ${FLATPAK_DEST}/share/sip --stubsdir=${FLATPAK_DEST}/lib/python3.9/site-packages"
+              ],
+              "dest-filename": "configure"
+            }
+          ],
+          "cleanup": [
+            "/bin",
+            "/include"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "enchant",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/AbiWord/enchant/releases/download/v2.3.1/enchant-2.3.1.tar.gz",
+          "sha256": "7b4b1afcf2cd8bfa691deea6188404d337f23174bbc39b9c2add2bf340736e9c"
+        }
+      ]
+    },
+    {
+      "name": "pandoc",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cp -R . /app/"
+      ],
+      "sources": [
+	    {
+          "type": "archive",
+          "url": "https://github.com/jgm/pandoc/releases/download/2.14.2/pandoc-2.14.2-linux-amd64.tar.gz",
```

Comment:
so you are building for x86_64 only here

(there is a pandoc tarball for aarch64 fortunately)

Reply:
How do I flag sources as to be used for aarch64 or x86_64 only then so that it will pick up the right one?

Reply:
You can add additional archive with URL to aarch64 to the sources array; then add `"only-arches": ["x86_64"]` or `"only-arches": ["aarch64"]` to each.

Reply:
This is fixed now with 30e2c42.